examples/zkapps/04-zkapp-browser-ui/ui/out/404.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><title>404: This page could not be found</title><meta name="next-head-count" content="3"/><link rel="preload" href="/04-zkapp-browser-ui/_next/static/css/ad2354abebf7054a.css" as="style"/><link rel="stylesheet" href="/04-zkapp-browser-ui/_next/static/css/ad2354abebf7054a.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/04-zkapp-browser-ui/_next/static/chunks/polyfills-78c92fac7aa8fdd8.js"></script><script src="/04-zkapp-browser-ui/_next/static/chunks/webpack-683523282bec2506.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/framework-305cb810cde7afac.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/main-15ee87a5f6d80118.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/pages/_app-7e7a1410ed9b0062.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/pages/_error-54de1933a164a1ff.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/IbSPwN67V1Yvix1r0yGn1/_buildManifest.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/IbSPwN67V1Yvix1r0yGn1/_ssgManifest.js" defer=""></script></head><body><div id="__next"><div style="font-family:system-ui,&quot;Segoe UI&quot;,Roboto,Helvetica,Arial,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center"><div style="line-height:48px"><style>body{color:#000;background:#fff;margin:0}.next-error-h1{border-right:1px solid rgba(0,0,0,.3)}@media (prefers-color-scheme:dark){body{color:#fff;background:#000}.next-error-h1{border-right:1px solid rgba(255,255,255,.3)}}</style><h1 class="next-error-h1" style="display:inline-block;margin:0 20px 0 0;padding-right:23px;font-size:24px;font-weight:500;vertical-align:top">404</h1><div style="display:inline-block"><h2 style="font-size:14px;font-weight:400;line-height:28px">This page could not be found<!-- -->.</h2></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":404}},"page":"/_error","query":{},"buildId":"IbSPwN67V1Yvix1r0yGn1","assetPrefix":"/04-zkapp-browser-ui","nextExport":true,"isFallback":false,"gip":true,"scriptLoader":[]}</script></body></html>
```

Comment:
```suggestion
<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><title>404: This page could not be found. Be sure to check Tutorial 4 for required configuration.</title><meta name="next-head-count" content="3"/><link rel="preload" href="/04-zkapp-browser-ui/_next/static/css/ad2354abebf7054a.css" as="style"/><link rel="stylesheet" href="/04-zkapp-browser-ui/_next/static/css/ad2354abebf7054a.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/04-zkapp-browser-ui/_next/static/chunks/polyfills-78c92fac7aa8fdd8.js"></script><script src="/04-zkapp-browser-ui/_next/static/chunks/webpack-683523282bec2506.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/framework-305cb810cde7afac.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/main-15ee87a5f6d80118.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/pages/_app-7e7a1410ed9b0062.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/pages/_error-54de1933a164a1ff.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/IbSPwN67V1Yvix1r0yGn1/_buildManifest.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/IbSPwN67V1Yvix1r0yGn1/_ssgManifest.js" defer=""></script></head><body><div id="__next"><div style="font-family:system-ui,&quot;Segoe UI&quot;,Roboto,Helvetica,Arial,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center"><div style="line-height:48px"><style>body{color:#000;background:#fff;margin:0}.next-error-h1{border-right:1px solid rgba(0,0,0,.3)}@media (prefers-color-scheme:dark){body{color:#fff;background:#000}.next-error-h1{border-right:1px solid rgba(255,255,255,.3)}}</style><h1 class="next-error-h1" style="display:inline-block;margin:0 20px 0 0;padding-right:23px;font-size:24px;font-weight:500;vertical-align:top">404</h1><div style="display:inline-block"><h2 style="font-size:14px;font-weight:400;line-height:28px">This page could not be found<!-- -->.</h2></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":404}},"page":"/_error","query":{},"buildId":"IbSPwN67V1Yvix1r0yGn1","assetPrefix":"/04-zkapp-browser-ui","nextExport":true,"isFallback":false,"gip":true,"scriptLoader":[]}</script></body></html>
```
```suggestion
<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><title>404: This page could not be found</title><meta name="next-head-count" content="3"/><link rel="preload" href="/04-zkapp-browser-ui/_next/static/css/ad2354abebf7054a.css" as="style"/><link rel="stylesheet" href="/04-zkapp-browser-ui/_next/static/css/ad2354abebf7054a.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/04-zkapp-browser-ui/_next/static/chunks/polyfills-78c92fac7aa8fdd8.js"></script><script src="/04-zkapp-browser-ui/_next/static/chunks/webpack-683523282bec2506.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/framework-305cb810cde7afac.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/main-15ee87a5f6d80118.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/pages/_app-7e7a1410ed9b0062.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/chunks/pages/_error-54de1933a164a1ff.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/IbSPwN67V1Yvix1r0yGn1/_buildManifest.js" defer=""></script><script src="/04-zkapp-browser-ui/_next/static/IbSPwN67V1Yvix1r0yGn1/_ssgManifest.js" defer=""></script></head><body><div id="__next"><div style="font-family:system-ui,&quot;Segoe UI&quot;,Roboto,Helvetica,Arial,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center"><div style="line-height:48px"><style>body{color:#000;background:#fff;margin:0}.next-error-h1{border-right:1px solid rgba(0,0,0,.3)}@media (prefers-color-scheme:dark){body{color:#fff;background:#000}.next-error-h1{border-right:1px solid rgba(255,255,255,.3)}}</style><h1 class="next-error-h1" style="display:inline-block;margin:0 20px 0 0;padding-right:23px;font-size:24px;font-weight:500;vertical-align:top">404</h1><div style="display:inline-block"><h2 style="font-size:14px;font-weight:400;line-height:28px">This page could not be found<!-- -->.</h2></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"statusCode":404}},"page":"/_error","query":{},"buildId":"IbSPwN67V1Yvix1r0yGn1","assetPrefix":"/04-zkapp-browser-ui","nextExport":true,"isFallback":false,"gip":true,"scriptLoader":[]}</script></body></html>
```

Reply:
not sure why two changes show here, my feedback is to enhance our 404 error message with some helpful guidance

Reply:
This 404 page is automatically generated by `NextJS`. I think it is a good idea to enhance the error message but we can create an issue and do it in a seperate pr.

Reply:
our new issue https://github.com/o1-labs/zkapp-cli/issues/470